scripts/src/javatests/org/oppia/android/scripts/maven/GenerateMavenDependenciesListTest.kt

```diff
@@ -569,9 +571,11 @@ class GenerateMavenDependenciesListTest {
     licenseName: String,
     verifiedLink: String,
   ) {
-    val extractedCopyLink = ExtractedCopyLink.newBuilder().setUrl(verifiedLink).build()
     assertThat(license.licenseName).isEqualTo(licenseName)
-    assertThat(license.extractedCopyLink).isEqualTo(extractedCopyLink)
+    assertThat(license.verifiedLinkCase).isEqualTo(
+      License.VerifiedLinkCase.EXTRACTED_COPY_LINK
+    )
+    assertThat(license.extractedCopyLink.url).isEqualTo(verifiedLink)
     assertThat(license.originalLink).isEqualTo(originalLink)
     assertThat(license.isOriginalLinkInvalid).isEqualTo(false)
```

Comment:
isFalse() here & elsewhere. Please make sure to check all places when comments say 'ditto' (or just in general--this is really good practice when following up to reviewer comments).

Reply:
Done.